src/main/java/de/chojo/repbot/listener/MessageListener.java

```diff
@@ -141,7 +141,7 @@ private void resolveNoTarget(Message message, GuildSettings settings) {
 
         for (var i = 0; i < members.size(); i++) {
             targets.put(requestEmojis[i], members.get(i));
-            (i % 2 == 0 ? first : second).add(requestEmojis[i] + " " + members.get(i).getAsMention());
+            (i <= members.size()/2 + members.size()%2 ? first : second).add(requestEmojis[i] + " " + members.get(i).getAsMention());
```

Comment:
```suggestion
            (i <= members.size() / 2 + members.size() % 2 ? first : second).add(requestEmojis[i] + " " + members.get(i).getAsMention());
```

Just make sure you have proper formatting ^.^ Rest looks fine ^-^

Reply:
Done ;-;